cnf-certification-test/lifecycle/suite.go

```diff
@@ -555,3 +565,24 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", podsWithRestrictedTolerationsNotDefault, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+func testStorageRequiredPods(env *provider.TestEnvironment) {
+	oc := clientsholder.GetClientsHolder()
+	var nonCompliantPods []*provider.Pod
+	for _, pod := range env.Pods {
+		for _, podOwner := range pod.ObjectMeta.GetOwnerReferences() {
+			if podOwner.Kind != statefulSet {
+				continue
+			}
```

Comment:
Looking at the requirement doc, I think we need to do _all_ pods, not just statefulSet ownerTypes.